templates/admin/judge/problem/change_form.html

```diff
@@ -11,10 +11,15 @@
 
 {% block after_field_sets %}{{ block.super }}
     {% if original %}
-        <a style="display: none" title="{% trans "View Submissions" %}" class="button submissions-link"
+        <a style="display: none" title="{{ _('View Submissions') }}" class="button submissions-link"
```

Comment:
iirc someone said to do this last review.

Reply:
You don't just blindly accept suggestions without checking to see if it works. If `_` works then you can keep it.

Reply:
It seems to work? The button works fine. Not sure what other checks I need.

Reply:
Well since it works, keep it.

Reply:
That's like saying you baked a cake successfully because it doesn't explode like a grenade...have you tried using the other languages?

Reply:
You're *almost* correct. Just as it would be rather exceptional for a cake you bake to have explosives in it, it would be really exceptional if `_` was an actual function usable in the template, yet not one of `gettext` or `gettext_lazy`.

This code is fine, both according to [the docs](https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#string-literals-passed-to-tags-and-filters) and [our code](https://github.com/DMOJ/online-judge/blob/master/templates/admin/judge/contest/change_form.html#L9)  with its resulting [translation string](https://github.com/DMOJ/online-judge/blob/master/locale/en/LC_MESSAGES/django.po#L3286).